src/network/connectionpool.py

```diff
@@ -41,9 +57,32 @@ def __init__(self):
         self.listeningSockets = {}
         self.udpSockets = {}
         self.streams = []
-        self.lastSpawned = 0
-        self.spawnWait = 2
-        self.bootstrapped = False
+        self._lastSpawned = 0
+        self._spawnWait = 2
+        self._bootstrapped = False
+
+        trustedPeer = BMConfigParser().safeGet('bitmessagesettings', 'trustedpeer')
+        try:
+            if trustedPeer:
+                host, port = trustedPeer.split(':')
+                self.trustedPeer = Peer(host, int(port))
```

Comment:
This could cause problems in the future if `trustedpeer` can be modified during runtime (now it can't so it should work correctly).

Reply:
Same here: I just moved this from `helper_startup`. Introduced in 0326315.